### DISCUSSION TITLE
After determinism what is next?

### DISCUSSION THREAD

**Discussion by ginxxxxx** [fishfolk/bones#128] [Q&A] [answered]
replay or netplay?

**Comment by erlend-sh** [comment]
Netplay. It’s already working, but it needs further optimization: https://github.com/fishfolk/jumpy/issues/790

**Reply by ginxxxxx** [reply]
could replay be the baseline for troubleshooting the netplay optimization in ticket https://github.com/fishfolk/jumpy/issues/790 ?

**Reply by zicklag** [reply]
Yes having a record and replay system could help us test network performance in a more automated way. Just to make it easier to kick off a demo network game without having to have multiple people ready to test play.

**Comment by zicklag** [comment, marked as answer]
Yeah, like @erlend-sh said, netplay is working in Jumpy, but it hasn't been standardized in bones yet, so if you are making your own game on bones, you have to setup the networking yourself.

The idea is that once we can we are going to start moving core pieces of Jumpy that work into a `bones_app` framework to help cover common use-cases like localization, networking, etc. for Bones + Bevy games.
